networkx/utils/backends.py

```diff
@@ -981,6 +994,10 @@ def _make_doc(self):
             else:
                 lines.append("")
 
+            if "backend_func_url" in func_info and func_info["backend_func_url"]:
+                lines.append(f"[`Source <{func_info['backend_func_url']}>`_]")
```

Comment:
What do you think of using the [walrus operator](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) here and elsewhere in this PR? For example:
```suggestion
            if func_url := func_info.get("backend_func_url"):
                lines.append(f"[`Source <{func_url}>`_]")
```